TwilioVerifySDK/TwilioVerify/Sources/Models/FactorPayload.swift

```diff
@@ -30,6 +30,8 @@ public protocol FactorPayload {
   var identity: String { get }
   ///Type of the factor.
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/c51c4513007d818dab4b64cd25c837959c7ec03d/TwilioVerifySDK/TwilioVerify/Sources/Models/FactorPayload.swift#L31"></span>Prefer at least one space after slashes for comments.<br />
<code>comment_spacing</code> <code>FactorPayload.swift:31</code></td>
    </tr>
  </tbody>
</table>